homeassistant/components/ecobee/climate.py

```diff
@@ -653,7 +679,11 @@ def set_temperature(self, **kwargs):
 
     def set_humidity(self, humidity):
         """Set the humidity level."""
+        if humidity not in range(0, 101):
+            _LOGGER.error("Invalid set_humidity value: %s", humidity)
```

Comment:
We can raise a ValueError with this message instead which will be shown in the frontend.

Reply:
No problem.